example_analysis/1.run_preprocessing_slurm.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#SBATCH --job-name=preprocessing   # Job name
+#SBATCH --partition=20                   # Partition name
+#SBATCH --ntasks=1                       # Run a single task
+#SBATCH --cpus-per-task=1               # Single CPU for the controller job
+#SBATCH --mem=4G                        # Memory for the controller job
+#SBATCH --time=72:00:00                 # Time limit (hrs:min:sec)
+#SBATCH --output=slurm_output/main/preprocessing-%j.out  # Standard output log
+
+# Set the path to the main Snakefile, the config file, and the workflow profile
+BRIEFLOW_PATH="../"
+CONFIG_FILE_PATH="config/config.yml"
+PROFILE_FILE_PATH="slurm/"
+
+# Create slurm output directories
+mkdir -p slurm_output/rule
+
+# Activate conda environment (adjust path as needed)
+source ~/.bashrc
+conda activate brieflow_workflows
+
+# Run Snakemake
+snakemake --use-conda --executor slurm \
+    --workflow-profile "$PROFILE_FILE_PATH" \
+    --snakefile "${BRIEFLOW_PATH}workflow/Snakefile" \
+    --configfile "$CONFIG_FILE_PATH" \
+    --until all_preprocess
```

Comment:
```suggestion
    --until all_preprocess
    
```

Following convention to [end files with new line](https://dev.to/documendous/why-you-should-end-your-source-files-with-a-new-line-156g#:~:text=One%20common%20rule%20is%20to,codebase%20remains%20clean%20and%20maintainable.)

Reply:
Changed in latest [commit](https://github.com/cheeseman-lab/brieflow/pull/18/commits/485b5e89ea72815b1fdb0a3c596c547425a9c232)